dataprofiler/tests/profilers/test_categorical_column_profile.py

```diff
@@ -160,13 +160,17 @@ def test_true_categorical_report(self):
         self.assertTrue(report["categorical"])
         six.assertCountEqual(
             self,
-            ['unique_count', 'unique_ratio', 'categories'], report['statistics']
+            ['unique_count', 'unique_ratio', 'categories', 'categorical_count'], report['statistics']
         )
         self.assertEqual(3, report["statistics"]["unique_count"])
         self.assertEqual(0.25, report["statistics"]["unique_ratio"])
         self.assertCountEqual(
             ["a", "b", "c"], report["statistics"]["categories"]
         )
+        expected_dict = {"a": 3, "b": 4, "c": 5}
+        self.assertDictEqual(expected_dict, report["statistics"]
+        ["categorical_count"])
+
```

Comment:
below in `test_false_categorical_report` we should probably change the test to validate the exact dict returned by the profile. In the current situation, it is not validating that `categorical_counts` is not in the dict. While we can do that, that would not future proof it like this this scenario. Instead if we specify the exact dict, it will prevent a potential future issue.

Reply:
below that, test_categorical_merge also needs updating like `test_false_categorical_report` such that it tests `categorical_counts` as well